vowpalwabbit/multiclass.cc

```diff
@@ -86,7 +91,8 @@ label_parser mc_label = {default_label, parse_label,
                          cache_label, read_cached_label,
                          delete_label, weight,
                          nullptr,
-                         sizeof(label_t)
+                         sizeof(label_t),
+                         get_int_label,
```

Comment:
Removable?

Reply:
Not sure how to get example label otherwise for different label types. They all stored in a union, that makes it a bit complicated.